sklearn/datasets/base.py

```diff
@@ -258,6 +258,15 @@ def load_iris():
 
     Read more in the :ref:`User Guide <datasets>`.
 
+    Parameters
+    ----------
+    
+        .. versionadded:: 0.18
```

Comment:
sorry i'm late to the party, shouldn't the `versionadded` be under `return_X_y`? and do we need a `versionadded` under the new return type (say around line 280)?